.env

```diff
@@ -0,0 +1 @@
+ELK_VERSION=6.8.0
```

Comment:
What does this file do? Where is it used? Are you sure that we should define versions like that? :thinking: